tests/components/sensor/test_recorder.py

```diff
@@ -0,0 +1,115 @@
+"""The tests for sensor recorder platform."""
+# pylint: disable=protected-access,invalid-name
+from datetime import timedelta
+from unittest.mock import patch, sentinel
+
+import pytest
+
+from homeassistant.components.recorder import history
+from homeassistant.components.recorder.const import DATA_INSTANCE
+from homeassistant.components.recorder.models import process_timestamp_to_utc_isoformat
+from homeassistant.components.recorder.statistics import statistics_during_period
+from homeassistant.setup import setup_component
+import homeassistant.util.dt as dt_util
+
+from tests.common import get_test_home_assistant, init_recorder_component
+from tests.components.recorder.common import wait_recording_done
+
+
+@pytest.fixture
```

Comment:
Can we import this fixture from the recorder tests ?

Reply:
Yes, but it trips pylint when fixtures are imported. The same fixture is used both by `recorder` and `history`, can it be moved to the root `conftest` instead?

Reply:
`components/conftest.py` maybe ?

Reply:
We have been importing fixtures before, just disable flake8/no-import-error is fine too.

Reply:
Probably nicer to keep the recorder fixture in the recorder/conftest.